lib/src/stream_channel_controller.dart

```diff
@@ -2,6 +2,9 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
+/// @docImport 'isolate_channel.dart';
```

Comment:
This is now available for use? Do we need a min. version of the sdk?

Reply:
I think we're only testing analyzer against `dev` – which is where it works 🤷